### ISSUE TITLE
My Account > Orders not mobile friendly

### ISSUE THREAD

**Issue by AimeeHapp** [AimeeHapp/live#23]
Hi Sergey, I wondered if you could work on something Flatsome related for me. My other developer about a year ago changed the Orders page around from the Flatsome default. On the computer it looks fine, but on the mobile, you have to scroll right to see the Access buttons (which are View buttons in Flatsome default). 

Is it possible to change the formatting ONLY for mobile? 

I'm attaching some screenshots of what it looks like on my mobile and what the Flatsome default is.

![](https://gitlab.com/happivize.com/happivize.com/uploads/c5ba28598eb8f85c749da76b87483481/image.png)

![](https://gitlab.com/happivize.com/happivize.com/uploads/912059a8b7c18e0fae87b263a985c472/image.png)

What I would like is if on MOBILE ONLY each order has the same info as computer, but is stretched over two lines so it would look like:

top row

Order # Date Status Button

second row
Product Name - $XX (full width of mobile)

So all the elements are the same just two lines for mobile. 

Is that possible?

**Comment by AimeeHapp** [comment]
Also, Aleksandr added this:

Template for this page wp-content/plugins/add-deliverables-info/public/partials/myaccount/orders.php. It will be helpful for Sergey

**Comment by shpp-shablyuk** [comment]
Is this orders page (https://dev.happivize.com/my-account/orders/)?

**Comment by AimeeHapp** [comment]
I can't see it from my phone but if you go to my account and click on the
orders button that's what I'm talking about

**Comment by shpp-shablyuk** [comment]
![screenshot 21](https://user-images.githubusercontent.com/13415218/48918648-aa08ab00-ee96-11e8-8b22-2fbaf6606949.png)

**Comment by AimeeHapp** [comment]
yes that looks like it

On Nov 22, 2018 1:39 PM, "shpp-shablyuk" <notifications@github.com> wrote:

[image: screenshot 21]
<https://user-images.githubusercontent.com/13415218/48918648-aa08ab00-ee96-11e8-8b22-2fbaf6606949.png>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/AimeeHapp/live/issues/23#issuecomment-441101702>, or mute
the thread

**Comment by shpp-shablyuk** [comment]
In your screenshot, the last column is ITEM. And on my screenshot - ACTIONS, and it is empty. It is not clear what should be in the "second row"

**Comment by AimeeHapp** [comment]
Hmm, you haven't changed anything? This is what it looks like on live and
dev was just refreshed last week

[image: image.png]

Let me ask Aleksandr

**Comment by shpp-shablyuk** [comment]
No, I did not change anything.

**Comment by AimeeHapp** [comment]
OK try now

**Comment by shpp-shablyuk** [comment]
The orders are empty
![screenshot 22](https://user-images.githubusercontent.com/13415218/48949568-0f18db00-ef41-11e8-98c5-37556668beea.png)

**Comment by AimeeHapp** [comment]
Yes that is it. I do not have any orders at the moment. If you did a test order of a free product like interview, then you will see it like I displayed in the screenshots in the initial post.

**Comment by shpp-shablyuk** [comment]
Ok
I keep working

**Comment by shpp-shablyuk** [comment]
Check, please
![screenshot 37](https://user-images.githubusercontent.com/13415218/49367317-66167100-f6f4-11e8-94c7-cc5cc1e89bb6.png)
.

**Comment by AimeeHapp** [comment]
Thanks, order with one item is close, but the last order with multiple items is confusing, glad you thought to test that. Here is a markup, I don't know if it's possible?

![image](https://user-images.githubusercontent.com/28787692/49390945-4aad6500-f6f8-11e8-9729-e8f29ef2a346.png)

**Comment by shpp-shablyuk** [comment]
Check, please.

![screenshot 45](https://user-images.githubusercontent.com/13415218/49513663-6a7c8e80-f89a-11e8-9a2d-187417ba7474.png)

**Comment by AimeeHapp** [comment]
Very close, can you make the Actions column less wide, the button looks so big. Make it 25% smaller.

Also I need to see an example with the Access button instead of View.  I don't know if you are on a local can you test buy  the product:  /product/test1 but first edit product and set price to $0 and then buy that, it will produce Access button. Just want to make sure it fits also.

**Comment by shpp-shablyuk** [comment]
Done
https://dev.happivize.com/my-account/orders/

![screenshot 46](https://user-images.githubusercontent.com/13415218/49518292-f1376880-f8a6-11e8-924c-50b62317b369.png)

**Comment by AimeeHapp** [comment]
Thanks that's looks good, but looks a little different on my screen, attaching image. Can you right justify the button, would that fix it?

![image](https://user-images.githubusercontent.com/28787692/49532416-6ea0b000-f88a-11e8-991e-dfc9b4db4bde.png)

**Comment by shpp-shablyuk** [comment]
Fixed. Check, please.

**Comment by AimeeHapp** [comment]
looks good, please provide files/instructions so i can upload to live.

**Comment by shpp-shablyuk** [comment]
You need to replace this file
`wp-content/plugins/add-deliverables-info/public/partials/myaccount/orders.php`
[orders.php.zip](https://github.com/AimeeHapp/live/files/2653507/orders.php.zip)


and add this styles to 
`wp-content/themes/flatsome-child/style.css`
[styles-for-orders.txt.zip](https://github.com/AimeeHapp/live/files/2653524/styles-for-orders.txt.zip)
